diff --git a/src/rockhopper/battery/CMakeLists.txt b/src/rockhopper/battery/CMakeLists.txt
index a296561..1b7821a 100644
--- a/src/rockhopper/battery/CMakeLists.txt
+++ b/src/rockhopper/battery/CMakeLists.txt
@@ -18,9 +18,9 @@
 
 include_directories(.)
 
-if(${WEBOS_TARGET_MACHINE_IMPL} STREQUAL emulator)
+# if(${WEBOS_TARGET_MACHINE_IMPL} STREQUAL emulator)
     nyx_create_module(BatteryMain fileutils.c batterylib.c emulator/fake_battery.c)
     install(FILES emulator/fake_battery_values.sh DESTINATION "/usr/sbin")
-elseif(${WEBOS_TARGET_MACHINE_IMPL} STREQUAL device)
-    nyx_create_module(BatteryMain fileutils.c batterylib.c device/battery.c)
-endif()
+# elseif(${WEBOS_TARGET_MACHINE_IMPL} STREQUAL device)
+#    nyx_create_module(BatteryMain fileutils.c batterylib.c device/battery.c)
+# endif()
